background.js

```diff
@@ -1,20 +1,16 @@
 function removeSafelink(requestDetails) {
-  if (requestDetails.url.includes('safelinks.protection.outlook.com/') ||
-    requestDetails.url.includes('statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html') ||
-    requestDetails.url.includes('outlook.office.com/mail/safelink.html')) {
-    var originalURL = getParameterByName('url', requestDetails.url);
+  var originalURL = getParameterByName('url', requestDetails.url);
 
-    console.debug('Blocked a "Microsoft Safe Link" redirect.')
+  console.debug('Blocked a "Microsoft Safe Link" redirect.')
 
-    return {
-      redirectUrl: originalURL
-    };
-  }
+  return {
+    redirectUrl: originalURL
+  };
 }
 
 browser.webRequest.onBeforeRequest.addListener(
   removeSafelink,
-  { urls: ["<all_urls>"] },
+  { urls: ["https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html*", "https://safelinks.protection.outlook.com/*", "https://outlook.office.com/mail/safelink.html*"] },
```

Comment:
This could be split up into a dedicated line for each URL, allowing for an easier extension if new URLs needed to be added:

```suggestion
  {
    urls: [
      "https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html*",
      "https://safelinks.protection.outlook.com/*",
      "https://outlook.office.com/mail/safelink.html*",
    ]
  },
```

I'll format this on `main`.

Reply:
I agree, this looks much better in separate lines. Thanks!!